bin/dev-restore-db

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env bash
+
+# DOC: Load a postgres dump into the dev database (pg_restore), replacing whatever state is currently in it. Takes a file name argument, file must be in current directory.
+
+pushd $(git rev-parse --show-toplevel)
+
+set -e
+set +x
+
+if [[ ! -f "$1" ]]; then
+  echo "Usage: bin/dev-restore-db DUMP_FILE_NAME"
+  exit 1
+fi
+
+# find repo root name (currently civiform), also support any names for backward and forward compatibility
+CIVIFORM_DIR=$(basename $(pwd))
+
+# set TRUNCATE_SQL to heredoc contents
+TRUNCATE_SQL=$(cat <<EOF
+TRUNCATE TABLE files;
+TRUNCATE TABLE applications CASCADE;
+TRUNCATE TABLE applicants CASCADE;
+TRUNCATE TABLE accounts CASCADE;
+TRUNCATE TABLE programs CASCADE;
+TRUNCATE TABLE questions;
+TRUNCATE TABLE versions;
+TRUNCATE TABLE versions_programs;
+TRUNCATE TABLE versions_questions;
+EOF
+)
+
+# remove newlines
+TRUNCATE_SQL=${TRUNCATE_SQL//$'\n'/}
+
+# location of dump file in docker container
+DUMP_FILE="project/$1"
```

Comment:
test if this exists

if [[ ! -f "$DUMP_FILE" ]]; then
    echo "$1 doesn't exist in the base directory."
    popd;
    exit 1;
fi

Reply:
already tested on L10. the `DUMP_FILE` path has `project` prepended to match the volume path in the docker container.

Reply:
ah missed that